### ISSUE TITLE
psycopg2 package rename

### ISSUE THREAD

**Issue by pkajaba** [openshiftio/openshift.io#3303]
This message keeps appearing in our logs:

`
/usr/lib64/python3.4/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
`

It means that package is going to rename in upcoming release. What we need to do is rename.

**Comment by humaton** [comment]
No, I have just upgraded to 2.7.4 to get over a bug in the previous release.
